packages/validator/src/commands/validateCmd.ts

```diff
@@ -0,0 +1,24 @@
+import * as utils from './utils';
+import { validate } from '../validator';
+import { Log, templates } from '../types';
+
+let template = process.env.npm_config_template || process.argv[2];
+const specs = process.env.npm_config_specs || process.argv[3];
+const version = process.env.npm_config_templateVersion || process.argv[4];
```

Comment:
The commands are very simple and yargs just seem just like overkill and not worth the effort in this case. Is it even possible for the commands to stay `yarn run validate ...`/`yarn run convert ...` instead of something like `yarn run validator validate ...`/`yarn run validator convert ...`? Either way in 962e3d8 this issue is fixed.

Reply:
I just took a look at yargs, it seems unnecessarily complicated to switch to yarn tbh. Using yargs might be useful in more complicated cli program, but validator is just a command with few arguments, which npm parses automatically...

Reply:
I'm pretty sure you won't be able to use named arguments instead of possitional ones without it. You're currently using npm for that but when the package is build and installed you're not using npm (nor yarn for that matter) to run it. It's run via `node` binary and that doesn't provide those variables. So, using yargs would also solve your npm vs. yarn problems.

Other than that, it provides the CLI with automatic help and a descriptive way to define arguments which is IMO better. It doesn't need to be in this PR but I would definitelly go this way since `npm_config_*` is a no go anyway.

Reply:
When used as a package it will call appropriate functions in `validator.ts` and `convertor.ts`, you can check how its used [here](https://github.com/api3dao/airnode/blob/f21eeb28b986360c39e7232a90bd49de53ae6d6d/packages/deployer/src/utils/files.ts#L11). Scripts in `commands/` directory are there only to call those functions based on what is in the arguments and in validator package they won't be used at all.

Reply:
Sorry, I didn't mean package as an included dependency I meant as a separate package installed basically as an executable with a CLI.

Reply:
I see, if yargs supports this then it should be integrated into validator, but in another PR perhaps